src/libraries/System.Reflection.Metadata/src/System/Reflection/Internal/Utilities/DecimalUtilities.cs

```diff
@@ -13,7 +13,12 @@ public static int GetScale(this decimal value)
 
         public static void GetBits(this decimal value, out bool isNegative, out byte scale, out uint low, out uint mid, out uint high)
         {
+#if NETCOREAPP
```

Comment:
How about the `GetBits` usage in the the `GetScale` method, above?

Reply:
Changed